cmd/oci/oci.go

```diff
@@ -239,7 +254,14 @@ func ProcessPlatformAndConfig(plat string, envName string) (hcl2nix.OCIArtifact,
 		return hcl2nix.OCIArtifact{}, "", fmt.Errorf("error: No such environment found. Valid oci environment that can be built are: %s", strings.Join(envNames, ", "))
 	}
 
-	return env, plat, nil
+	return artifact, plat, nil
+}
+
+func verifyBase(a hcl2nix.OCIArtifact) error {
+	if a.Environment == "pkgs" && !a.Base {
```

Comment:
let's rename `Environment` variable everywhere to `Artifact` maybe?

Reply:
Cool